magic/legality.py

```diff
@@ -17,6 +17,17 @@ def legal_formats(d: Container, formats_to_check: Set[str] = None, errors: Dict[
     if errors is None:
         errors = {}
     formats_to_discard = set()
+
+    # When the formatting of the deck cannot be read, the decklist reader will return an empty deck.
+    # When the text box is empty, a different error will pop up.
+    # So if the text box is not empty, but no cards are read, it must be a formatting issue.
+    if (sum(e['n'] for e in d.maindeck) + sum(e['n'] for e in d.sideboard)) <= 0:
+        for f in formats_to_check:
+            add_error(errors, f, 'Legality_General', 'Deck\'s formatting cannot be read.')
```

Comment:
The style checker is dinging you here because it wants `"Deck's formatting cannot be read."` not `'Deck\'s formatting cannot be read.'`. You can see errors like this ahead of time via `dev.py lint` (see also `dev.py release` and `dev.py test`).

Let's say something a bit more friendly like: "I'm afraid I don't recognize that decklist format. Try exporting from MTGO."

Reply:
Ok, I'll fix it.

Reply:
For the Linter, I should be able to soon figure out a way for VS Code to automatically Lint stuff.
For now I'll just manually edit things.